tdvt/tdvt/tdvt.py

```diff
@@ -107,7 +107,8 @@ def __init__(self, test_set: TestSet, test_config: TdvtInvocation, lock, verbose
         self.verbose = verbose
         self.thread_lock = lock
         self.temp_dir = make_temp_dir([self.test_config.suite_name, str(thread_id)])
-        self.test_config.output_dir = self.temp_dir
+        if (self.test_config.output_dir == ''):
```

Comment:
With the user-specified directory it seems like we're missing:
- creating it if its missing
- the bucketing by suite_name and thread_id with the current temp_dir design
Do those suite_name/thread_id need to be appended to the user-specified one?

Reply:
@rosswbrown is right about `temp_dir` issues, I did some testing and there are some unexpected behaviors.